finite_difference/src/parallel/parallel_utils_stub_mod.f90

```diff
@@ -27,21 +27,28 @@
 !------------------------------------------------------------------------------
 ! Author: A. R. Porter, STFC Daresbury Laboratory
 
-!> Stub implementation to be used in place of parallel_mod when NOT
+!> Stub implementation to be used in place of parallel_utils_mod when NOT
 !! compiling with MPI
-module parallel_mod
-  use parallel_common_mod, only: get_rank, get_num_ranks,      &
-                                 set_proc_grid, get_proc_grid, &
-                                 nranks, rank, nprocx, nprocy
+module parallel_utils_mod
+  use kind_params_mod, only: go_wp
   implicit none
 
   private
 
+  !> MPI rank + 1 of current process
+  integer :: rank
+  !> Total no. of MPI processes
+  integer :: nranks
+
+  integer, parameter :: MPI_UNDEFINED = -99
```

Comment:
Afaik MPI_UNDEFINED etc are part of the MPI standard, we should likely just avoid using MPI_ as prefix anywhere.

Reply:
That's a good point. I've created some parameters with different names and expose them instead.